plugins/@grouparoo/pipedrive/src/lib/export/organizations/destinationMappingOptions.ts

```diff
@@ -0,0 +1,169 @@
+import {
+  DestinationMappingOptionsMethod,
+  DestinationMappingOptionsResponseType,
+  SimpleAppOptions,
+  objectCache,
+} from "@grouparoo/core";
+
+import { connect } from "../../connect";
+import { PipedriveClient } from "../../client";
+import { GROUP_FIELD_PREFIX } from "./listMethods";
+
+export interface PipedriveCacheData {
+  appId: string;
+  appOptions: SimpleAppOptions;
+}
+
+export const destinationMappingOptions: DestinationMappingOptionsMethod =
+  async ({ appOptions, appId }) => {
+    const client = await connect(appOptions);
+    const allKnown = await getKnownOrganizationFields(
+      client,
+      { appId, appOptions },
+      true
+    );
+
+    const required = allKnown.filter((field) => field.required);
+
+    // Filter out required fields and those that we create to manage group membership
+    const knownNotRequired = allKnown.filter(
+      (field) => !field.required && !field.groupMembershipField
+    );
+
+    return {
+      labels: {
+        property: {
+          singular: "Pipedrive Field",
+          plural: "Pipedrive Fields",
+        },
+        group: {
+          singular: "Pipedrive Filter",
+          plural: "Pipedrive Filters",
+        },
+      },
+      properties: {
+        known: knownNotRequired,
+        required,
+        allowOptionalFromProperties: false,
+      },
+    };
+  };
+
+const mapTypesToGrouparoo = (
+  pipedriveType: string,
+  fieldName: string
+): DestinationMappingOptionsResponseType => {
+  // https://pipedrive.readme.io/docs/core-api-concepts-custom-fields#section-types-of-custom-fields
+
+  const overrides: Record<string, DestinationMappingOptionsResponseType> = {
+    Name: "string",
+  };
+
+  if (overrides[fieldName]) return overrides[fieldName];
+
+  const map: Record<string, DestinationMappingOptionsResponseType> = {
+    varchar: "string",
+    varchar_auto: "string",
+    text: "string",
+    double: "number",
+    monetary: "number",
+    enum: "integer",
+    user: "integer",
+    org: "integer",
+    people: "integer",
+    phone: "phoneNumber",
+    date: "date",
+    address: "string",
+    daterange: "date",
+
+    // TODO
+    time: null, // time in the format of "00:00:00"
```

Comment:
@pauloouriques  is mapping many of these to string. I think we can do that here too for `time`

Reply:
By looking to this function
```typescript
function formatVar(value) {
  if (value === undefined) {
    return null;
  }

  if (value instanceof Date) {
    return value.toISOString();
  }

  return value;
}
```

Mapping dates are applicable because there is a type `Date`, in our case here, how to detect time values??

Reply:
We don;t have the type of "time" but making this a string will allow the user to come up with their own string property in the right format and be able to set the remote field.